RetailSuiteAccounting/Troubleshooting/01_HandleNonReleasableSupplierReceipt.md

```diff
@@ -0,0 +1,73 @@
+[!!User Interface Supplier receipts](../UserInterface/01_Book.md#supplier-receipts)
+[!!Post a supplier receipt](../Operation/10_ManageReceipts.md#post-a-supplier-receipt)
+
+# Handle a non-releasable supplier receipt
+
+Check and, if necessary, modify the supplier settings to release a supplier receipt in the *Purchasing* module.
+
+#### Error Description
+
+When trying to release a supplier receipt, the *No items linked* window with the notice *Please link items first* is displayed in the *CHECK SUPPLIER RECEIPTS* tab in the *Check supplier receipts* menu entry of the *Purchasing* module.
+
+Follow the instructions below to check and modify the supplier settings in the *Accounting* module.
+
+#### Prerequisites
+
+- A supplier receipt has been entered in the system, see [Acquire a supplier receipt](../Operation/10_ManageReceipts.md#acquire-a-supplier-receipt).
+- The supplier receipt to be released is displayed in the *Check supplier receipts*, see [Display a missing supplier receipt](./02_DisplayMissingSupplierReceipt.md).
+
+[comment]: <> (Diese Prerequisites machen hier vielleicht nicht so viel Sinn, denn beide sind Voraussetzungen, damit man überhaupt auf das Error kommt. So lassen oder No prerequisites to fulfill?)
+
+#### Procedure
+
+*Accounting > Post > Button EDIT > Menu entry Supplier receipts > Tab Overview*
+
+  ![Overview](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsOverview01.png "[Overview]")
+
+1. Check the supplier number for the non-releasable supplier receipt in the *Overview* tab and switch to the *Customer/supplier* tab: *Accounting > Settings > DEBTORS/CREDITORS*.    
```

Comment:
Check and note?
Deutlicher machen, dass du die Supplier number für die nachfolgenden Schritte noch brauchst.